internal/model/label.go

```diff
@@ -0,0 +1,14 @@
+package model
+
+// Label represents a label shown in the Marketplace UI.
+type Label struct {
+	Name        string `json:"name"`
+	Description string `json:"description"`
+	URL         string `json:"url"`
+}
+
+var OfficialLabel Label = Label{
+	Name:        "Official",
+	Description: "This plugin is maintained by Mattermost",
+	URL:         "https://mattermost.com/pl/default-community-plugins",
```

Comment:
This will need to point to the docs. We need to set up the link before the 5.20 release.